src/main/java/csv/WriteCSV.java

```diff
@@ -0,0 +1,36 @@
+package main.java.csv;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteCSV {
+
+    private final static String PEOPLE_DATA = "people.csv";
+
+    public static void main(String[] args) {
+        PersonCSV header = new PersonCSV("Name", "Age", "City");
+        String headerString = header.PersonToString(header);
+
+        PersonCSV person1 = new PersonCSV("Maga", "22", "Kyiv");
+        String personString1 = person1.PersonToString(person1);
+
+        PersonCSV person2 = new PersonCSV("Sasha", "21", "Kyiv");
+        String personString2 = person2.PersonToString(person2);
+
+        PersonCSV person3 = new PersonCSV("Timoshka", "5", "Kyiv");
+        String personString3 = person3.PersonToString(person3);
+
+        try (FileWriter writer = new FileWriter(PEOPLE_DATA)) {
+
+            writer.write(headerString + "\n");
+            writer.write(personString1 + "\n");
+            writer.write(personString2 + "\n");
+            writer.write(personString3 + "\n");
+
+            System.out.println("Данные успешно записаны в файл " + PEOPLE_DATA);
```

Comment:
What if some developer from England, Germany or India will check your code...
He will not understand this text...